website/addons/googledrive/utils.py

```diff
@@ -95,7 +95,7 @@ def serialize_settings(node_settings, current_user):
         path = node_settings.folder_path
         if path is not None:
             ret['currentPath'] = '/' + path.lstrip('/')
-            ret['currentFolder'] = '/ (Full Google Drive)' if path == '/' else '/' + path
+            ret['currentFolder'] = '/ Full Google Drive' if path == '/' else '/' + path
```

Comment:
See previous comment re: display name. I didn't mean to suggest removing the parentheses.